modules/mw.cx.init.Translation.js

```
@@ -85,6 +85,18 @@ mw.cx.init.Translation.prototype.init = function () {
 
 			this.sourceWikiPage.setRevision(sourcePageContent.revision);
 
+			if (this.sourceWikiPage.getLanguage() == "mdwiki") {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)